routers/repo/http.go

```diff
@@ -627,7 +637,7 @@ func GetTextFile(p string) func(*context.Context) {
 			h.setHeaderNoCache()
 			file := ctx.Params("file")
 			if file != "" {
-				h.sendFile("text/plain", "objects/info/"+file)
+				h.sendFile("text/plain", "objects/info/"+path.Clean("/" + file)[1:])
```

Comment:
```suggestion
				h.sendFile("text/plain", "objects/info/"+path.Clean(file))
```

Reply:
```suggestion
				h.sendFile("text/plain", "objects/info/"+file)
```

Leave it as before, let `sendFile` handle all cases correctly.